pkg/kubelet/kubelet.go

```diff
@@ -1468,8 +1468,10 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 	go kl.volumeManager.Run(kl.sourcesReady, wait.NeverStop)
 
 	if kl.kubeClient != nil {
-		// Start syncing node status immediately, this may set up things the runtime needs to run.
```

Comment:
for the context on this comment: https://github.com/kubernetes/kubernetes/pull/14542#discussion_r40460183